### PR TITLE
Module 10

### PR DESCRIPTION
**Pull request by josephfernandes**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

File: Dockerfile
Comment (JackMeadDev) [inline Dockerfile:26]: Note that this changes your behaviour slightly so that the app will error if the user doesn't specify a PORT environment variable; so your Readme instructions will now fail! Adding in a default value, e.g. with the below, would help by keeping your instructions consistent but still allowing Azure to override the PORT if/when it wants to
```suggestion
ENV PORT=5000
CMD poetry run gunicorn todo_app.app:app -b 0.0.0.0:$PORT
```

File: .github/workflows/my-ci-pipline.yml
Comment (JackMeadDev) [inline .github/workflows/my-ci-pipline.yml:23]: Note that I don't think you've made the changes to your Azure App Service directly either, so we'll want to do that once the CD is set up (or even just make them directly when merging this!)

File: todo_app/data/mongodb.py
Comment (JackMeadDev) [inline todo_app/data/mongodb.py:1]: Now that this file exists, it would be good to clear out `trello_items.py` and any other Trello related content since it's no longer relevant

File: todo_app/data/mongodb.py
Comment (JackMeadDev) [inline todo_app/data/mongodb.py]: We definitely don't want these lines! You'll be printing your connection string - which includes a DB password - to the console and logging it in Azure; we should get rid of this

File: todo_app/data/viewmodel.py
Comment (JackMeadDev) [inline todo_app/data/viewmodel.py:19]: Nice work bringing these back 🙂

File: README.md
Comment (JackMeadDev) [inline README.md]: Thanks for adding these. We should also update the required environment variables, for example:
* Adding a note here on which variables to add to the `.env` file
* Adding the relevant keys (but not the values) to the `.env.template` file

**Review by JackMeadDev — APPROVED** [review]
Nice work, I was easily able to run your app and manage my to-dos in a Mongo/Cosmos DB now! I've left a few optional suggestions below although the most important one is to remove your `print(key)` statement that is currently logging your DB password out! Once that's done, I'm happy for you to merge this when you're ready

File: todo_app/data/mongodb.py
Comment (josephfernandes) [reply todo_app/data/mongodb.py:1]: Will do

**Pull request merged**
